packages/pyright-internal/src/analyzer/packageTypeVerifier.ts

```diff
@@ -99,7 +94,7 @@ export class PackageTypeVerifier {
             this._configOptions.evaluateUnknownImportsAsAny = true;
         }
 
-        this._execEnv = this._configOptions.findExecEnvironment('.');
+        this._execEnv = this._configOptions.findExecEnvironment(Uri.file('.'));
```

Comment:
Should `Uri.file(".")` be a constant? Do we use it elsewhere?

Reply:
This might be the only spot. But it is effectively a constant because of the caching that happens.